WalnutExternal.lua

```diff
@@ -3,20 +3,62 @@
 VULKAN_SDK = os.getenv("VULKAN_SDK")
 
 IncludeDir = {}
-IncludeDir["VulkanSDK"] = "%{VULKAN_SDK}/Include"
+IncludeDir["VulkanSDK"] = "%{VULKAN_SDK}/include"
 IncludeDir["glm"] = "../vendor/glm"
 
 LibraryDir = {}
-LibraryDir["VulkanSDK"] = "%{VULKAN_SDK}/Lib"
+LibraryDir["VulkanSDK"] = "%{VULKAN_SDK}/lib"
 
 Library = {}
-Library["Vulkan"] = "%{LibraryDir.VulkanSDK}/vulkan-1.lib"
+
+if os.ishost("linux") and os.istarget("linux") then
+   Library["Vulkan"] = "%{LibraryDir.VulkanSDK}/libvulkan.so.1"
```

Comment:
All that is needed is `links { "vulkan" }` for the compiler to link in the Vulkan library. If it were the case that the Vulkan library were installed in a non-standard directory, only then would it be required to specify that additional directory.

Reply:
The best what you can do is to apply your changes on your tree, commit them, push the new branch to your clone of Walnut on github and open a new merge request, referencing #22 and #11.

I don't have any capacity currently to continue working on it, just commit your changes and open a new pull request.